tests/phpunit/tests/theme/wpThemeJson.php

```diff
@@ -3493,4 +3493,331 @@ function test_get_styles_for_block_with_content_width() {
 		$style_rules = $theme_json->get_styles_for_block( $metadata );
 		$this->assertSame( $expected, $root_rules . $style_rules );
 	}
+
+	/**
+	 * @dataProvider data_generate_spacing_scale_fixtures
+	 *
```

Comment:
Based on feedback on the other PRs, for consistency with the other tests, I believe we need to add a `@ticket` line to these doc comments, so that it's linked to the trac ticket for the backports:

```
 * @ticket 56467
```